lld/test/COFF/lto-cache-errors.ll

```diff
@@ -0,0 +1,20 @@
+; REQUIRES: x86
```

Comment:
https://maskray.me/blog/2021-08-08-toolchain-testing#the-test-checks-at-the-wrong-layer

A better test is llvm-lto2 --cache-dir in `llvm/test/ThinLTO/X86`

Reply:
My first iteration was there - but it doesn't seem to follow the same code path, I never got the error message that I got from lld-link.

I will merge this now, since a test is better than no test - but I might follow up with a test in the llvm layer when I do the changes to report_fatal_error if people think that's a good change.